dbux-projects/src/dataLib/PathwaysDataProvider.js

```diff
@@ -312,27 +312,25 @@ export default class PathwaysDataProvider extends DataProviderBase {
     if (!lastStep || action.newStep) {
       return true;
     }
-    if (StepType.is.Other(stepType)) {
-      if ((applicationId && applicationId !== lastApplicationId) ||
-        (staticContextId && staticContextId !== lastStaticContextId)
-      ) {
-        return true;
+
+    if (StepType.is.None(stepType)) {
```

Comment:
I'm not sure if this will work. I would replace it with `if (!stepType) { ...`

Reply:
We've change `StepType.None` to 1 in some PR before, since type with value 0 breaks the assumption and cause some error elsewhere. Also Steps should always has a `type`, I think we don't need to check that here.